packages/base/Container/src/Container/Container.tsx

```diff
@@ -101,6 +101,8 @@ export const Container: ContainerProps = documentable(
       } = props
 
       const spacingProps = { gap, padded, top, bottom, right, left }
+      const isBorderedVariant =
+        !variant || variant === 'white' || variant === 'transparent'
```

Comment:
```suggestion
        variant === 'white' || variant === 'transparent'
```
we don't add a border when the variant is not defined, or do we?

Reply:
we currently add a border when the variant is not defined. Is the `With default border` example defined in storybook.